src/main.rs

```diff
@@ -11,24 +20,22 @@ fn main() -> io::Result<()> {
         return Ok(());
     }
 
-    let mut input_start = 1;
-
     let mut pangram = "abcdefghijklmnopqrstuvwxyz";
 
-    if args[1] == "-p" || args[1] == "--pangram" {
-        pangram = args[2].as_str();
-        input_start = 3;
+    if let Some(p) = matches.get_one::<String>("pangram") {
+        pangram = p;
     }
 
+    println!("Pangram: {}", &pangram);
```

Comment:
I'm not totally sure if we want this to display, given the user should already know what the pangram is based on input.